src/cf-atomic-component/src/utilities/on-ready/index.js

```diff
@@ -0,0 +1,49 @@
+/* ==========================================================================
+   On Ready
+
+   Utility for triggering functions only after the page is loaded
+
+   ========================================================================== */
+
+'use strict';
+
+const _readyFunctions = [];
+
+/**
+ * Checks if the document is ready, if it is, trigger the passed function,
+ * if not, push the function to an array to be triggered after the page
+ * is loaded.
+ * @param {Function} fn -
+ *   Function to run only after the DOM has completely loaded.
+ * @returns {Array} List of functions to run after the DOM has loaded.
+ */
+function onReady( fn ) {
```

Comment:
Outside scope, but could we change this to accept any elem, so it's useful whether we're waiting on the doc or an individual piece of the page?

Reply:
Hmm, so it takes a dom node and waits till just that dom node is ready? I like it!

Reply:
Yeah, it's the way jQuery ready works